aces/Loris
# [Raisinbread] Change Visit Stage to match Instrument data

Changed the Visit stage from `Pass` to `In Progress` for the Candidate with `CandID=587630` and `sessionID=1053` because some instruments are still incomplete. I will be checking for other similar inconsistencies in the Raisinbread data in this PR as well. 

#### Testing instructions

With a fresh RB install:
1. Navigate to `https://your-loris-url/instrument_list/?candID=587630&sessionID=1053`
2. See that the Visit Status is now set to `In Progress`

#### Link to related issue

* Resolves #5684
